### PR TITLE
condor_ce_info_status: safely handle bad data

### PR DESCRIPTION
**Pull request by matyasselmeci**
Don't crash if entries are received from the collector with missing values for `CPUs` or `Memory`. Also handle if the attribute for `CPUs` is actually `Cpus`. This is for SOFTWARE-2185.

### CODE REVIEW

File: src/condor_ce_info_status [hunk 1]
```diff
@@ -14,6 +14,17 @@ import htcondor
 
 import htcondorce.info_query
 
+# Add htcondor.param.get() to old versions of condor-python that don't have it
+try:
+    htcondor.param.get
```
Comment (bbockelm) [inline src/condor_ce_info_status]: When was `param.get` introduced - and what's the minimum version of HTCondor supported by the CE?

I suspect we can simply drop the workaround.

File: src/condor_ce_info_status [hunk 2]
```diff
@@ -44,9 +55,9 @@ def formatResourceAdsTable(resources, width):
     table += "\n"
     for resource in resources:
         catalog_entry = resource.catalog_entry
-        table += "\n" + format_str % (catalog_entry['Name'],
-                                      catalog_entry['CPUs'],
-                                      catalog_entry['Memory'],
+        table += "\n" + format_str % (catalog_entry.get('Name','???'),
+                                      catalog_entry.get('CPUs', catalog_entry.get('Cpus', '???')),
```
Comment (bbockelm) [inline src/condor_ce_info_status]: Is `catalog_entry` a ClassAd object?  If so, lookup should be case-insensitive.

File: src/condor_ce_info_status (hunk 1 above)
Comment (matyasselmeci) [inline src/condor_ce_info_status]: Yeah, I think it was for 8.2 compatibility. I'll drop it.

File: src/condor_ce_info_status (hunk 2 above)
Comment (matyasselmeci) [inline src/condor_ce_info_status]: It should be in theory, but isn't in practice. I mentioned this in the ticket.

**Pull request merged**